dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/LagomPlugin.scala

```diff
@@ -244,10 +248,30 @@ object LagomReloadableService extends AutoPlugin {
       ).map(LagomReloadableServiceCompat.joinAnalysis)
     }.value,
 
+    lagomVerifyProjectFamilyAlignment := Def.task {
+
+      val evalState = state.value
+      val logger = evalState.log
+      logger.info("Verifying Project Family Alignment on [${thisProjectRef.value.project}]")
+      val deps = (dependencyClasspath in Compile).value
+      val dependencyClassLoader = new java.net.URLClassLoader(deps.files.map(_.toURI.toURL).toArray)
```

Comment:
I wish I could just reuse the classloader used at compile time. Couldn't find a way to get a grab to it. Maybe @dwijnand has a good tip on this.

The advantage of the check we have in Akka is that it uses the runtime classloader.